### PR TITLE
chore: update embedding sample to be more developer friendly

### PR DESCRIPTION
**Pull request by jackwotherspoon**
Note: I am a Googler on a different team that noticed this sample was slightly confusing when I tried to use it so putting this PR up as a potential improvement. 

As a developer wanting to copy/paste this sample into my workload I would expect it to take in an input string and then get the embedding for it? This is what the notebook example for this already does... https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/generative_ai/text_embedding_new_api.ipynb

### CODE REVIEW

File: generative_ai/embedding.py [hunk 1]
```diff
@@ -16,13 +16,12 @@
...
+def text_embedding(text: str = "What is life?") -> list:
     """Text embedding with a Large Language Model."""
     model = TextEmbeddingModel.from_pretrained("textembedding-gecko@001")
-    embeddings = model.get_embeddings(["What is life?"])
-    for embedding in embeddings:
-        vector = embedding.values
-        print(f"Length of Embedding Vector: {len(vector)}")
+    embeddings = model.get_embeddings([text])
+    vector = embeddings[0].values
+    print(f"Length of Embedding Vector: {len(vector)}")
```
Comment (jackwotherspoon) [inline generative_ai/embedding.py:24]: removed loop here, really no need to loop if we are only returning 1 embedding...

**Comment by jackwotherspoon** [comment]
@yil532 curious what you think on this? (saw you as a previous contributor to this file) 😄

**Comment by jackwotherspoon** [comment]
FYI: Test that is failing has nothing to do with code touched in this PR

**Review by gericdong — APPROVED** [review]

**Review by m-strzelczyk — APPROVED** [review]

**Comment by jackwotherspoon** [comment]
Looks like test that was blocking this PR was fixed as part of https://github.com/GoogleCloudPlatform/python-docs-samples/pull/11313

**Pull request merged**